src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar-box">
+      <div class="calendar calendar--start-day--wed calendar--month-length_30">
+        <div class="calendar__day calendar__day--first"></div>
```

Comment:
no-no-no, don't add modifiers for n-th elements, use structural pseudo-classes instead
```suggestion
        <div class="calendar__day"></div>
```